src/renderer/components/tree-view/tree-label-edit.js

```diff
@@ -19,20 +19,25 @@ export default class LabelEditor extends Component {
   };
 
   state = {
-    title: 'Untitled'
+    title: 'Untitle'
```

Comment:
Why is **Untitled** changed to **Untitle**?

Reply:
oh, this is a typo! I fix it.